engine/pkg/orm/client.go

```diff
@@ -100,6 +100,7 @@ type JobClient interface {
 	UpdateJob(ctx context.Context, jobID string, values model.KeyValueMap) error
 	DeleteJob(ctx context.Context, jobID string) (Result, error)
 
+	GetJobByIDWithSoftDelete(ctx context.Context, jobID string) (*frameModel.MasterMeta, error)
```

Comment:
Can we add a `CreateJob` method directly? `CreateJob` uses an `INSERT` statement to create a record that is able to detect the existing soft deleted job. It can also solve the atomic issue.

Reply:
Use `INSERT` is better, have addressed, ptal